lib/diagnostic.rb

```diff
@@ -89,7 +127,17 @@ def example
     Example.first
   end
 
-at  describe 'DELETE destroy' do
-    # your test(s) here
+  describe 'DELETE destroy' do
+    before(:each) do
+      delete :destroy, id: example, format: :json
```

Comment:
Just make sure you're sending `example.id` rather than `example`. Tests are great otherwise!